modules/channels.js

```diff
@@ -0,0 +1,32 @@
+var Channels = `
+                        <div class="image_holder" onclick="openImage('gallery/19.jpg')">
```

Comment:
The indentation here looks like the `HTML` code was pasted in a hurry.

Can you try to fix this? I'll recommend 2 spaces per indent.

Reply:
okay sir